src/main/java/com/courtcanva/ccfranchise/dtos/FranchiseeDto.java

```diff
@@ -0,0 +1,22 @@
+package com.courtcanva.ccfranchise.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class FranchiseeDto {
+
+    private String businessName;
+
+    private StaffDto staff;
+
+    private Integer  ABN;
+
+    private String businessAddress;
```

Comment:
are you sure we only want to use one column to store one address?

Reply:
use state and postcode  and business address